src/components/project/project.service.ts

```diff
@@ -1200,4 +1243,25 @@ export class ProjectService {
       ).every((n) => n)
     );
   }
+
+  protected async validateOtherResourceId(
+    ids: string[],
```

Comment:
This could be
```suggestion
    ids: Many<string>,
```
allowing a single item or a list to be passed in. So we don't have to wrap them in lists if they aren't.

It could be converted to a list below like this:
```tsx
for (const id of many(ids)) {
```